### ISSUE TITLE
How to ignore some instances of the force cast rule

### ISSUE THREAD

**Issue by Tableau-David-Potter** [realm/SwiftLint#145]
I have some cases where I want to use a force cast - e.g. when it would be a programming error if the force failed.  One example is loading a view controller from a storyboard.  Is there any way to ignore specific violations of a rule?

Thanks,
***David***

**Comment by keith** [comment]
You can wrap your valid cases in comments:

```swift
// swiftlint:disable force_cast
let foo = bar as! Bad
// swiftlint:enable force_cast
```

**Comment by Tableau-David-Potter** [comment]
Great, that works.  Thanks!

**Comment by keith** [comment]
No problem, we'll update the readme soon!
